### PR TITLE
Arf pool migrate

### PR DESCRIPTION
**Pull request by shan-57blocks**

### CODE REVIEW

File: packages/huma-web-shared/src/hooks/usePoolV2.ts [hunk 1]
```diff
@@ -636,6 +636,52 @@ export function useCancellableRedemptionInfoV2(
...
+    poolName,
+    trancheType,
+    provider,
+  )
+  const [refreshCount, refresh] = useForceRefresh()
+
+  useEffect(() => {
+    if (account && vaultContract) {
+      const fetchData = async () => {
+        const redemptionRecords = await vaultContract.lenderRedemptionRecords(
```
Comment (ljiatu) [inline packages/huma-web-shared/src/hooks/usePoolV2.ts:663]: nit: should be `redemptionRecord`.

File: packages/huma-web-shared/src/hooks/usePoolV2.ts [hunk 2]
```diff
@@ -636,6 +636,52 @@ export function useCancellableRedemptionInfoV2(
...
+          account,
+        )
+        const cancellableRedemptionShares =
+          await vaultContract.cancellableRedemptionShares(account)
+        const cancellableRedemptionAssets = await vaultContract.convertToAssets(
+          cancellableRedemptionShares,
+        )
+        setRedemptionStatus({
+          numSharesRequested: redemptionRecords.numSharesRequested.toString(),
+          withdrawableAssets: redemptionRecords.totalAmountProcessed.toString(),
```
Comment (ljiatu) [inline packages/huma-web-shared/src/hooks/usePoolV2.ts:673]: This should be `totalAmountProcessed - totalAmountWithdrawn`. FWIW, there is a `withdrawablAssets` view function.

File: packages/huma-web-shared/src/hooks/usePoolV2.ts (hunk 1 above)
Comment (shan-57blocks) [reply packages/huma-web-shared/src/hooks/usePoolV2.ts]: Updated

File: packages/huma-web-shared/src/hooks/usePoolV2.ts (hunk 2 above)
Comment (shan-57blocks) [reply packages/huma-web-shared/src/hooks/usePoolV2.ts]: Updated

**Review by ljiatu — DISMISSED** [review]
LGTM but @mliu please double check.

**Review by mliu — DISMISSED** [review]

**Review by mliu — APPROVED** [review]

**Pull request merged**